org.eclipse.xtext.testing/tests/org/eclipse/xtext/testing/tests/junit5/RunJunit5Suite.java

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Sebastian Zarnekow and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.xtext.testing.tests.junit5;
+
+import org.junit.runner.RunWith;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.platform.suite.api.SelectPackages;
+
+@RunWith(JUnitPlatform.class)
+@SelectPackages("org.eclipse.xtext.testing.tests.junit5")
+public class RunJunit5Suite {
```

Comment:
what is this class good for?

Reply:
are tests executed ion gradle at all?

Reply:
Doesn't look like these are being executed at all. I renamed the Suite to Tests. Let's see